src/components/pages/index.js

```diff
@@ -22,6 +22,20 @@ function Index({ initialState }) {
   const currentHomeLanguage = homeLanguages[currentHomeLanguageIndex];
   const [isSearching, setIsSearching] = useState(false);
 
+  const handlePageShow = event => {
+    if (event.persisted) {
+      window.location.reload(true);
+    }
+  };
+
+  React.useEffect(() => {
+    window.addEventListener('pageshow', handlePageShow);
+
+    return () => {
+      window.removeEventListener('pageshow', handlePageShow);
```

Comment:
I have some feeling about this not working as it should let me check please.

Reply:
That's what the PR is here for

Reply:
Yep, this doesn't get fired at all, but I don't understand why the event listener is not registered on the other pages. Hmmmmmmmmm

Reply:
Why would it be on the other pages, when it's deregistered when you leave this page? 🤷

Reply:
The problem is that it doesn't deregister, it works the same if you don't do anything

Reply:
It doesn't as we call `document.location`, so the component is never unmounted, nevertheless it's good practice to hve it there, I find

Reply:
Ok, true, this doesn't get called in this case. But it gets "detached" by default with the reload anyway.